app/views/statistics/index.html.erb

```diff
@@ -1,6 +1,12 @@
 <% title _("Statistics"), _('Statistics') %>
-<div id="statistics">
-<% charts.each do |ch| %>
-  <%= content_tag(:div, ch, :class=>"stats-well") %>
-<% end %>
-</div>
+
+<div id="statistics"></div>
+
+<%= javascript_tag defer: 'defer' do -%>
```

Comment:
I've refactored this to be more reusable. I'm not sure about creating a helper. I'll look into it further.